sql/index/pilosa/driver.go

```diff
@@ -79,6 +79,15 @@ type (
 func NewDriver(root string) *Driver {
 	h := pilosa.NewHolder()
 	h.Path = filepath.Join(root, "."+DriverID)
+
+	if err := h.Open(); err != nil {
+		panic(err)
```

Comment:
we shouldn't panic here

Reply:
yeah, shall we return an error from the constructor (I don't like neither)

Reply:
If it can fail, it should return an error